packages/app-web/src/views/app/app.tsx

```diff
@@ -68,13 +68,15 @@ export default function AppView (): JSX.Element {
 
   useWindowLoadListener(onAppLoad)
 
-  // Shortcut handler
+  /**
+   * Shortcut handler, receives shortcut notation from a function that calls it, this function looks up the shortcut notation in the shortcut "lookup" table and dispatches the action
+   */
   const onShortcut = useCallback((shortcut: string, event: KeyboardEvent) => {
     const actions = shortcutBindings[shortcut]
     if (actions !== undefined) {
       if (typeof actions === 'string') {
         dispatch({ type: actions, payload: {} })
-      } else {
+      } else { // Allow for also dispatching a whole bunch of actions
```

Comment:
Move this comment into an own line below this one. Also, I'd call it 'multiple actions at the same time' instead of 'a whole bunch'.

Reply:
Ok